server/models/annotation.py

```diff
@@ -436,8 +491,21 @@ def load(self, id, region=None, getElements=True, *args, **kwargs):
             annotation.
         :returns: the matching annotation or none.
         """
+        force = kwargs.pop('force', False)
+        kwargs['force'] = True
```

Comment:
I think this would be clearer if `force` was explicitly in the method parameter list, and the call to `annotation  = super(...).load()` included `force=True` instead of modifying `kwargs`.